lib/src/smtp/internal_representation/ir_header.dart

```diff
@@ -24,16 +83,26 @@ abstract class _IRHeader extends _IROutput {
     // At least 10 chars (random length).
     if (availableLength < 10) availableLength = 10;
 
-    var splitData = split(convert.utf8.encode(value), availableLength);
+    var second = false;
+    for (var d in split(convert.utf8.encode(value), availableLength)) {
+      if (second) yield _$eolSpace;
```

Comment:
was there a bug in my code?

Reply:
No, I just refactored it a bit so I can reuse some code and constants.

Reply:
I can't find the `yield _$eolSpace` in my old code.

But your refactoring looks good to me.  (If it was in my code good.  If not it will be added by your code)